AutolayoutPractice.xcodeproj/project.pbxproj

```diff
@@ -268,6 +269,28 @@
 		};
 /* End PBXResourcesBuildPhase section */
 
+/* Begin PBXShellScriptBuildPhase section */
```

Comment:
[Suggestion] This definitely works for the initial project. I would actually recommend you moving swiftlint itself to be ran in a bash script. There are some cases in which you may want to move all linters to a separate step or run linting outside of building the project.

Reply:
The bash script would live in a `/scripts` folder which would run from the home directory of your xcode project